cosmos/carbon.json

```diff
@@ -132,19 +132,19 @@
       "coinDecimals": 18,
       "coinGeckoId": "binance-usd",
       "gasPriceStep": {
-        "low": 76923076923.07692,
-        "average": 96153846153.84616,
-        "high": 115384615384.61539
+        "low": 33333333333.333333333333,
+        "average": 33333333333.333333333333,
+        "high": 33333333333.333333333333
       }
     },
     {
       "coinDenom": "CGLP",
       "coinMinimalDenom": "cglp.1.19.1698d3",
       "coinDecimals": 18,
       "gasPriceStep": {
-        "low": 76923076923.07692,
-        "average": 96153846153.84616,
-        "high": 115384615384.61539
+        "low": 33333333333.333333333333,
+        "average": 33333333333.333333333333,
+        "high": 33333333333.333333333333
```

Comment:
Same issue as comment on L104, need to round ceil, if not user will be paying insufficient gas. Also, follow the original number, round to 5 d.p.